common/buildcraft/lib/misc/StackMatchingPredicate.java

```diff
@@ -0,0 +1,10 @@
+package buildcraft.lib.misc;
```

Comment:
One last thing: can you add the buildcraft license header to the top of these two files? (Which is currently at the top of StackUtil)